app/api/v1/mediator/election.py

```diff
@@ -76,7 +75,12 @@ def create_election(
         election_id = str(uuid4())
 
     key_ceremony = get_key_ceremony(data.key_name, request.app.state.settings)
-    context = CiphertextElectionContext.from_json_object(data.context)
+    try:
+        context = data.context.to_sdk_format()
+    except ValueError as e:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST, detail=str(e)
```

Comment:
This is actually incorrect. This should be using the `repr()` since this is for developers not `str()`